dimod/higherorder.py

```diff
@@ -133,7 +135,10 @@ def make_quadratic(poly, strength, vartype=None, bqm=None):
 
 
 def _reduce_degree(bqm, poly, vartype, scale):
+    """
+    helper function of make_quadratic
 
+    """
```

Comment:
Kind of redundant I think. Could be one line `"""Helper function for make_quadratic"""`.

google styleguide on docstring:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods
another example:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html